open_prices/users/models.py

```diff
@@ -33,9 +33,14 @@ class Meta:
         verbose_name = "User"
         verbose_name_plural = "Users"
 
+    @property
```

Comment:
@raphodn This is important addition, there was a bug here, because we just checked that the `User` instance had a `is_authenticated` method when doing `if user.is_authenticated` (which is always true)

Reply:
this requires a dedicated "fix" PR :pray: 
and to be tested, because it's probably here for a reason ? (that I can't remember) :thinking:

Reply:
The method is here because we use it during proof upload. It was a hidden bug, as we used to block all non authenticated requests (but not anymore), so the user was always authenticated.